include/quic/connection.hpp

```diff
@@ -150,6 +150,9 @@ namespace oxen::quic
         //		dcid: remote CID used for this connection
         //      path: network path used to reach remote client
         //      ctx: IO session dedicated for this connection context
+        //      alpns: passed directly to TLS session for handshake negotiation
```

Comment:
Are these specified in best->worst order, or worst->best?  (This probably digs down into ngtcp2 land, but would be nice to figure that out and document it here).

Reply:
just checked, server chooses the first one on the client's list that it also supports.

Reply:
thanks Tom, i wasn't sure off the top of my head. added